plugins/groovy/src/org/jetbrains/plugins/groovy/lang/completion/GrMethodOverrideCompletionProvider.java

```diff
@@ -0,0 +1,106 @@
+/*
```

Comment:
I don't really know what to test here, since suggestion candidates are provided by `OverrideImplementExploreUtil.getMethodsToOverrideImplement(psiClass, implement)` and then filtered by IDEA. I could try testing if it should suggest some methods on different places (class definition, body definition, etc.), but to be honest I don't really know how to do that.